packages/core/src/repositories/evaluationResultsRepository/index.ts

```diff
@@ -104,7 +101,7 @@ export class EvaluationResultsRepository extends Repository<
           eq(evaluations.workspaceId, this.workspaceId),
         ),
       )
-      .as('evaluationResultsBaseQuery')
+      .$dynamic()
```

Comment:
this is the key. This forces this query to be part of whatever other query it's embedded into, preventing the use of subqueries which hinder performance.